docs/queryworkers/sql/advanced-commands.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 10
+title: Advanced SQL Queries
+---
+
+Case-sensitivity is supported for attribute names using `"<attribute name>"`.
+Case-sensitive collections names can be used as well.
+
```

Comment:
@dlozina-macrometa, please add headings before each example to help the user understand what it is about. This also builds a side menu on the right, which you will see if you do a local build. Headings help users scan and navigate documentation.

Reply:
👍

Reply:
I will add the headings as you proposed @diana-macrometa. It makes a lot of sense. Sorry for the delay, I just lost this from my radar. 🙏

Reply:
Ups I actually did this but have not push 🥲